spec/test_app/test_suite.sh

```
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 
+set -euo pipefail
+
 echo "UID=${UID}" >&2
 echo "PS4=${PS4}" >&2
 
 cd $(dirname $0)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)